test/fixtures/rules/std_rule_cases/L052.yml

```diff
@@ -180,55 +274,93 @@ test_fail_same_line_multiple_inline_comment:
 
 test_pass_newline_inline_comment:
   pass_str: |
-    SELECT a FROM foo -- inline comment
+    SELECT a
+    FROM foo -- inline comment
     ;
   configs:
     rules:
       L052:
-        semicolon_newline: True
+        multiline_newline: True
 
 test_fail_newline_inline_comment:
   fail_str: |
-    SELECT a FROM foo -- inline comment
+    SELECT a
+    FROM foo -- inline comment
     
     ;
   fix_str: |
-    SELECT a FROM foo -- inline comment
+    SELECT a
+    FROM foo -- inline comment
     ;
   configs:
     rules:
       L052:
-        semicolon_newline: True
+        multiline_newline: True
 
-test_fail_newline_multiple_inline_comments:
+test_fail_newline_multiple_inline_comments_custom_oneline:
   fail_str: |
     SELECT a FROM foo -- inline comment
```

Comment:
Nitpick: The two identical comments initially confused me. What would you think about changing them to something like:
```
-- inline comment #1
-- inline comment #2
```

Reply:
This is a good point, better to explicitly show that there's no crazy switching round of comment positions going on 👍